Stripe/PublicHeaders/STPPaymentContext.h

```diff
@@ -267,6 +267,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, strong) UIView *addCardViewControllerFooterView;
 
+/**
+ If the value of this property is true, STPPaymentMethodsViewController creates
+ a card source instead of a card token when your user adds a card. The default
+ value is false.
+
+ @see https://stripe.com/docs/sources/cards#create-source
+ */
+@property (nonatomic, assign) BOOL createsCardSources;
```

Comment:
This almost feels to me like it should be part of the `STPPaymentConfiguration`. wdyt?

Reply:
definitely! i'll move it